hadoop-hdds/server-scm/src/main/java/org/apache/hadoop/hdds/scm/ha/BackgroundSCMService.java

```diff
@@ -126,7 +126,6 @@ public void stop() {
         log.info("{} Service is not running, skip stop.", getServiceName());
         return;
       }
-      backgroundThread.interrupt();
```

Comment:
Why remove this statement?

Reply:
When current stopping thread calls interrupt, its interrupted and further handling is stopped throwing interrupt exception. This is causing to stop to fail.